R/set_pc_rownames.R

```diff
@@ -0,0 +1,23 @@
+#' Label rownames of provided PCs and remove NA batch cells from PCs
+#'
+#' @param integrated_pcs The PCs with unlabeled rownames
```

Comment:
```suggestion
#' @param pcs The PCs with unlabeled rownames
```
This doesn't have to be integrated.